src/pages/faucet.tsx

```diff
@@ -1,23 +1,12 @@
 import { ReactElement } from 'react'
-import { useRouter } from 'next/router'
 import Verify from '../components/Faucet'
-import AssetProvider from '@context/Asset'
 import content from '../../content/pages/faucet.json'
 import Page from '@components/@shared/Page'
 
 export default function PageVerify(): ReactElement {
-  const router = useRouter()
-  const { did } = router.query
-
   return (
-    <AssetProvider did={did as string}>
-      <Page
-        title={content.title}
-        description={content.description}
-        uri={router.route}
-      >
-        <Verify didQueryString={did as string} />
-      </Page>
-    </AssetProvider>
+    <Page title={content.title} description={content.description} uri="">
+      <Verify />
```

Comment:
issue: rename import to `Faucet`

Reply:
This will be addressed in a subsequent pull request.